custom/enikshay/reports/repeaters.py

```diff
@@ -183,19 +184,27 @@ def headers(self):
 
     @property
     def rows(self):
-        vouchers = [CommCareCase.wrap(flatten_result(result)) for result in self._search_results().raw_hits]
+        return self.get_rows(paged=True)
+
+    @property
+    def get_all_rows(self):
+        return self.get_rows(paged=False)
+
+    def get_rows(self, paged=True):
+        vouchers = [
+            CommCareCase.wrap(flatten_result(result))
+            for result in self._search_results(paged=True).raw_hits
```

Comment:
Shouldn't this pass through the given value of `paged`?

Reply:
welp. don't know how I missed that. 🤦‍♂️